hypha/templates/includes/login_button.html

```diff
@@ -2,23 +2,68 @@
 
 {% if request.user.is_authenticated %}
     {% if request.path == "/" and request.user.can_access_dashboard %}
-        <a
-            class="button button--transparent button--narrow {{ class }}"
-            href="{{ APPLY_SITE.root_url }}{% url 'dashboard:dashboard' %}"
-        >
-            {% heroicon_micro "user" class="inline align-text-bottom w-4 h-4 me-1" aria_hidden=true %}
-
-            {% blocktrans %}My {{ ORG_SHORT_NAME }}{% endblocktrans %}
-        </a>
+        <div x-data="{ show: false }" class="relative flex">
+            <button x-on:click="show = ! show" class="button--narrow font-bold text-center flex items-center justify-center px-2 hover:bg-black hover:text-white active:bg-black active:text-white focus:bg-black focus:text-white" type="button">
```

Comment:
Add a "button" class here. The adds the small runded corners we have on all buttons now.